CONTRIBUTING.md

```diff
@@ -7,16 +7,16 @@ There are a few things to consider before contributing to open-feature-operator.
 Firstly, there's [a code of conduct](https://github.com/open-feature/.github/blob/main/CODE_OF_CONDUCT.md).
 TLDR: be respectful.
 
-Any contributions are expected to include tests. These can be validated with `make test` or the automated github workflow will run them on PR creation.
+Any contributions are expected to include tests. These can be validated with `make unit-test` or the automated github workflow will run them on PR creation.
 
 The go version in the `go.mod` is the currently supported version of go.
 
 Thanks! Issues and pull requests following these guidelines are welcome.
 
 ## Development
 
-### FeatureFlagConfiguration custom resource definition versioning
-Custom resource definitions support multiple versions. The kubebuilder framework exposes a system to seamlessly convert between versions (using a "hub and spoke" model) maintaining backwards compatibility. It does this by injecting conversion webhooks that call our defined convert functions. The hub version of the `FeatureFlagConfiguration` custom resource definition (the version to which all other versions are converted) is `v1alpha1`.
+### FeatureFlag custom resource definition versioning
+Custom resource definitions support multiple versions. The kubebuilder framework exposes a system to seamlessly convert between versions (using a "hub and spoke" model) maintaining backwards compatibility. It does this by injecting conversion webhooks that call our defined convert functions. The hub version of the `FeatureFlag` custom resource definition (the version to which all other versions are converted) is `v1beta1`.
```

Comment:
Should "our" be more specific?  Like either "Keptn" or "FeatureFlag"?  And what about a link to these definitions?

Apparently this file is not subject to markdownlint but, if it were, it's going to fail because of this long line with multiple sentences in one line.  Just for your consideration.

I guess this file is not subject to

Reply:
the aim of this PR was to move from v1alpha1 to v1beta1. It's not that important to restructure and improve the docs here, that can be done in a follow-up PR

Reply:
@StackScribe It would be great if you could give our docs a general review, but ya, this is focused on a particular update.

cc @beeme1mr